plugins/spu2-x/src/Windows/Config.cpp

```diff
@@ -205,6 +205,35 @@ void WriteSettings()
     DebugConfig::WriteSettings();
 }
 
+void CheckOutputModule(HWND window)
+{
+    OutputModule = SendMessage(GetDlgItem(window, IDC_OUTPUT), CB_GETCURSEL, 0, 0);
+    const bool IsConfigurable =
+        mods[OutputModule] == PortaudioOut ||
+        mods[OutputModule] == WaveOut ||
+        mods[OutputModule] == DSoundOut;
+
+    const bool AudioExpansion =
+        mods[OutputModule] == XAudio2Out ||
+        mods[OutputModule] == PortaudioOut;
+
+    EnableWindow(GetDlgItem(window, IDC_OUTCONF), IsConfigurable);
+
+    EnableWindow(GetDlgItem(window, IDC_SPEAKERS), AudioExpansion);
+}
+
+void UpdateVolumeSlider(HWND hWnd)
```

Comment:
I don't think moving this into a function really improves anything.

Reply:
Done.